client/lib/analytics/index.js

```diff
@@ -412,13 +412,10 @@ const analytics = {
 			}
 		},
 
-		recordPageView: function( urlPath, pageTitle ) {
-			if ( ! isGoogleAnalyticsAllowed() ) {
-				return;
-			}
-
-			analytics.ga.initialize();
-
+		recordPageView: makeSafeDebuggableGoogleAnalyticsFunc( function recordPageView(
+			urlPath,
+			pageTitle
+		) {
```

Comment:
I find it a bit confusing that the `function` is named. Have you considered using arrow notation?

Reply:
That was intentional to make sure we have `func.name`, which makes the debugging output more informative:

https://github.com/Automattic/wp-calypso/blob/08d999c76af71b20b3d4c361594ef58156f25bd5/client/lib/analytics/index.js#L522-L525

That way we can write

```
[Disallowed] analytics recordEvent( /* args */ )
```

And you can inspect more closely _what_ GA event was fired with what args.